server/routes/shared_api/autocomplete/helpers.py

```diff
@@ -69,6 +70,7 @@ def execute_maps_request(query: str, language: str) -> Dict:
       'language': language
   }
   response = requests.post(MAPS_API_URL + urlencode(request_obj), json={})
+  logging.info("Executed Google Maps Prediction API request.")
```

Comment:
wonder if we need this because if the request fails, this should throw an error & you have another logging in place to count the number of predictions

Reply:
done.